### DISCUSSION TITLE
Azure dev ops git based content

### DISCUSSION THREAD

**Discussion by craftzneko** [StaticJsCMS/static-cms#581] [General]
Hi, looking to migrate from netlify CMS now it seems to be EOL. Has anybody used static-cms with azure dev-ops hosted git content, is it possible?

**Comment by KaneFreeman** [comment]
Static CMS does not support Azure. Netlify CMS does support it though AND it has just been picked up by another company (See https://www.netlify.com/blog/netlify-cms-to-become-decap-cms/).

**Reply by craftzneko** [reply]
Talk about a timely announcement. I quite like Static cms though, any plans to consider supporting an azure dev ops backend?

**Reply by KaneFreeman** [reply]
Not at present. There are 4 supported backends (GitHub, GitLab, Gitea, Bitbucket) already and with me being the only Maintainer on this project I cannot take on any more at current.

**Reply by KaneFreeman** [reply]
Out of curiosity what about Static CMS (assuming it supported Azure) would make you want to use it over Netlify CMS? I am planning on contributing to the Netlify CMS (actually talking to the new owners about it at the moment) and kind of curious what you like about Static CMS that maybe I could look into adding to Netlify CMS.

**Reply by craftzneko** [reply]
Tbh i had not set it up yet and i was looking at screenshots for another product 😂. I love netlify cms and i am really glad its going to get some love!
